src/main/java/nextstep/fp/Lambda.java

```diff
@@ -26,31 +26,19 @@ public void run() {
         }).start();
     }
 
+    public static int sumAllByCondition(List<Integer> numbers, Conditional conditional) {
+        return numbers.stream().filter(conditional::test).reduce(0, Integer::sum);
```

Comment:
```suggestion
        return numbers.stream()
                .filter(conditional::test)
                .mapToInt(Integer::intValue)
                .sum();
```
mapToInt(), sum()을 사용하면 이런 방법으로도 사용해볼 수 있어요. :)